cmake/project-config.cmake.in

```diff
@@ -12,8 +12,21 @@ include(CMakeFindDependencyMacro)
 
 set(SLS_USE_HDF5 "@SLS_USE_HDF5@")
 
-# Add optional dependencies here
+
+find_package(ZeroMQ 4 QUIET)
```

Comment:
Shouldn't we use `find_dependency` here?

Reply:
Using this construct we need find_package, instead of the find_dependency wrapper (https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html) otherwise the QUIET keyword is not respected and instead the options passes to the original find_package call is used.

Reply:
Oh I see.